deploy/gke/redaction/README.md

```diff
@@ -68,4 +68,105 @@ When you are done, you can clean up everything you've done with the following st
 
 ```
 kubectl delete namespace $OTEL_NAMESPACE
-```
\ No newline at end of file
+```
+
+## Try it out yourself (Optional)
+
+This section contains steps that you can follow to actually see redaction you setup earlier in action. Since there is no expected outcome from our kubernetes deployment due to a lack of telemetry data, the redaction processor setup in our collector cofiguration is not applied to anything. In this section we will introduce a source of telemetry data to actually see the redaction in action.  
+
+### Prerequisites 
```

Comment:
For each example (e.g. redaction here), there should be one prerequisites section at the top.  Then, users should be able to follow the example from top-to-bottom.  If there are additional overall prerequisites, list them at the top.  If there are additional verification steps, add those as verification step sections in the readme.

Reply:
Makes sense, I was thinking to remove this heading and just merging its content under "Try it out yourself" heading.